src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/JmapSendMessagesScenario.scala

```diff
@@ -6,17 +6,19 @@ import org.apache.james.gatling.jmap.scenari.common.CommonSteps
 import org.apache.james.gatling.jmap.{HttpSettings, JmapMessages}
 
 import scala.concurrent.duration._
+import org.apache.james.gatling.control.{Username, Password, User}
 
 class JmapSendMessagesScenario extends Simulation {
 
   val userCount = 100
-  val feeder = UserFeeder.createUserFeederWithInboxAndOutbox(userCount)
+  val users = UserFeeder.createCompletedUserFeederWithInboxAndOutbox(userCount)
```

Comment:
what if we create a function that return the array for users and we map it to a feeder when we need it ?
It sounds more logical, don't you think ?